### PR TITLE
[WIP]set mountopt for ubuntu

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/main.go [hunk 1]
```diff
@@ -31,11 +33,22 @@ func main() {
 		// had a specific job to do as a subprocess, and it's done.
 		return
 	}
-
+	f, err := ioutil.TempFile("/tmp", "pprof-podman")
+	if err != nil {
+		fmt.Println(err.Error())
+		os.exit(1)
```
Comment (zhangguanzhang) [inline cmd/podman/main.go:39]:
```suggestion
		os.Exit(1)
```

**Comment by rhatdan** [comment]
@baude can we close this one, I don't think this is still being worked on?

**Comment by zhangguanzhang** [comment]
any update?

**Comment by vrothberg** [comment]
I am taking a shot and close the PR. I assume it was opened while analyzing some CI slowness on Ubuntu which has been addressed in the meantime by @baude.

**Pull request closed without merging**